### PR TITLE
[feat] Add isConnected API

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- add `isConnected` API.
- rename `open` -> `connect`, `close` -> `disconnect` API.

### CODE REVIEW

File: src/ainize.ts [hunk 1]
```diff
@@ -22,10 +23,15 @@ export const request = async <T>(
   ainize: Ainize,
   { serviceName, opType, data, timeout = DEFAULT_TIMEOUT_MS }: AinizeRequest
 ): Promise<AinizeResponse<T>> => {
+  if (!Handler.getInstance().isConnected()) {
+    throw new Error('Network error: blockchain event channel is not connected.');
```
Comment (jiyoung-an) [inline src/ainize.ts:27]: cc: @platfowner @changyeop1

File: src/ainft.ts [hunk 2]
```diff
@@ -151,9 +152,9 @@ export default class AinftJs {
   }
 
   /**
-   * Opens a session with the blockchain endpoint.
+   * Connects to the blockchain endpoint.
    */
-  async open() {
+  async connect() {
```
Comment (platfowner) [inline src/ainft.ts:157]: how about adding some checking logic e.g. if (...isConnected()) {...}?

File: src/ainft.ts [hunk 3]
```diff
@@ -163,9 +164,17 @@ export default class AinftJs {
...
    */
-  async close() {
+  isConnected(): boolean {
+    return Handler.getInstance().isConnected();
+  }
+
+  /**
+   * Disconnects from the blockchain endpoint.
+   */
+  async disconnect() {
```
Comment (platfowner) [inline src/ainft.ts:177]: ditto?

Comment (jiyoung-an) [reply src/ainft.ts:179]: thanks!

File: src/ainft.ts [hunk 4]
```diff
@@ -151,22 +152,34 @@ export default class AinftJs {
...
+   * Connects to the blockchain endpoint.
    */
-  async open() {
-    const privateKey = this.ain.wallet.defaultAccount?.private_key;
-    if (privateKey) {
-      await this.ainize.login(privateKey);
-    } else {
-      await this.ainize.loginWithSigner();
+  async connect() {
+    if (!this.isConnected()) {
```
Comment (platfowner) [inline src/ainft.ts:158]: how about throwing some exception for the other case?

File: src/ainft.ts [hunk 5]
```diff
@@ -151,22 +152,34 @@ export default class AinftJs {
...
+    return Handler.getInstance().isConnected();
+  }
+
+  /**
+   * Disconnects from the blockchain endpoint.
    */
-  async close() {
-    await this.ainize.logout();
+  async disconnect() {
+    if (this.isConnected()) {
```
Comment (platfowner) [inline src/ainft.ts:180]: ditto?

File: src/ainft.ts (hunk 4 above)
Comment (jiyoung-an) [reply src/ainft.ts]: okay, done!

File: src/ainft.ts (hunk 5 above)
Comment (jiyoung-an) [reply src/ainft.ts]: okay, done!

File: src/ainft.ts [hunk 6]
```diff
@@ -151,9 +152,12 @@ export default class AinftJs {
   }
 
   /**
-   * Opens a session with the blockchain endpoint.
+   * Connects to the blockchain endpoint.
    */
-  async open() {
+  async connect() {
+    if (this.isConnected()) {
+      throw new Error('Client is already connected.');
```
Comment (platfowner) [inline src/ainft.ts:159]: 👍

File: src/ainft.ts [hunk 7]
```diff
@@ -163,9 +167,20 @@ export default class AinftJs {
...
+    return Handler.getInstance().isConnected();
+  }
+
+  /**
+   * Disconnects from the blockchain endpoint.
    */
-  async close() {
+  async disconnect() {
+    if (!this.isConnected()) {
+      throw new Error('Client is not connected.');
```
Comment (platfowner) [inline src/ainft.ts:182]: 👍

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by jiyoung-an** [comment]
Thank you for quick review @platfowner! now merging...

**Pull request merged**